### PR TITLE
updating linux references throughout all docs, to DD

### PR DESCRIPTION
**Pull request by aevesdocker**

### CODE REVIEW

File: buildx/working-with-buildx.md [hunk 1]
```diff
@@ -16,13 +16,7 @@ multiple nodes concurrently.
...
 ### Windows and macOS
 
-Docker Buildx is included in [Docker Desktop](../desktop/index.md) for Windows
-and macOS.
-
-### Linux packages
-
-Docker Linux packages also include Docker Buildx when installed using the
-[DEB or RPM packages](../engine/install/index.md).
+Docker Buildx is included in [Docker Desktop](../desktop/index.md) for Windows, macOS, and Linux.
```
Comment (thaJeztah) [inline buildx/working-with-buildx.md:19]: Should we keep a separate heading for Linux installations (so a heading for "Docker Desktop on Windows, macOS, Linux", and one for Linux?)

Comment (aevesdocker) [reply buildx/working-with-buildx.md]: Have pushed another commit to fix this. Do my headings work?

**Comment by aevesdocker** [comment]
The following page also mentions Linux - https://docs.docker.com/compose/install/

@dockertopia is covering this in a separate PR.

**Review by usha-mandya — APPROVED** [review]
LGTM. Thank you @aevesdocker

**Pull request merged**